tests/checkpoint/test_checkpoint_with_fluent_datasources.py

```diff
@@ -1196,15 +1150,15 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
     # create expectation suite
     context.add_expectation_suite("my_expectation_suite")
 
-    batch_request_0: FluentBatchRequest = FluentBatchRequest(
-        datasource_name="my_sqlite_datasource",
-        data_asset_name="table_partitioned_by_date_column__A_query_asset_limit_5",
-    )
+    batch_request_0 = {
+        "datasource_name": "my_sqlite_datasource",
+        "data_asset_name": "table_partitioned_by_date_column__A_query_asset_limit_5",
+    }
 
-    batch_request_1: FluentBatchRequest = FluentBatchRequest(
-        datasource_name="my_sqlite_datasource",
-        data_asset_name="table_partitioned_by_date_column__A_query_asset_limit_10",
-    )
+    batch_request_1 = {
```

Comment:
This is the correct change, due to the type requirements in the `Checkpoint` configuration.